src/backend/metal/Cargo.toml

```diff
@@ -28,8 +28,9 @@ name = "gfx_device_metal"
 [dependencies]
 log = "0.3"
 gfx_core = { path = "../../core", version = "0.5" }
-cocoa = "0.3"
+cocoa = "0.2"
```

Comment:
There was some version mismatch between metal bindings and the one used in the gfx backend, didn't feel like pushing a new version so I took the lazy way out